config/config.json.example

```diff
@@ -1,16 +1,30 @@
-{
-    "moderator_channel": "moderation-logs",
-    "support_channel": "support",
-    "rules_channel": "rules",
-    "non_support_channels": [
-        "general"
-    ],
-    "bad_words": [
-        "bum",
-        "hole"
-    ],
-    "appeal_url": "https://www.google.com",
-    "client_id": "",
-    "client_secret": "",
-    "client_token": ""
-}
+{
+    "moderator_channel": "moderation-logs",
+    "support_channel": "support",
+    "rules_channel": "rules",
+    "non_support_channels": [
+        "general"
+    ],
+    "moderated_channels": [
+        "general",
+        "support"
+    ],
+    "bypass": {
+        "roles": [
+            "ATLauncher Staff",
+            "NodeCraft Staff",
+            "Moderators",
+            "AllmightyBot"
+        ],
+        "users": [
+		]

```

Comment:
This looks to be indented one too short

Reply:
users is under bypass next to roles, their square brackets are parallel. I believe it is correct. jsonlint finds no problem with this

Reply:
correction: when viewed in the online editor it shows as aligned, however in the viewer, it is one off. will investigate

Reply:
was two tabs, is now spaces